binder/environment.yml

```
@@ -0,0 +1,13 @@
+name: example-environment
+channels:
+  - conda-forge
+dependencies:
+  - jupyter-server-proxy
+  - matplotlib-base
+  - nodejs
+  - numpy
+  - pip
+  - python
+  - xeus-python
+  - pip:
+    - vega-datasets
```

It looks like `vega-datasets` is also available on conda forge: https://anaconda.org/conda-forge/vega_datasets

```suggestion
  - vega_datasets
  - xeus-python
```